auth/jwt.go

```diff
@@ -0,0 +1,70 @@
+package auth
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	abc "github.com/dgrijalva/jwt-go"
+	"github.com/go-chi/jwtauth/v5"
+)
+
+var TokenAuth *jwtauth.JWTAuth
+
+type User struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+func init() {
+	TokenAuth = jwtauth.New("HS256", []byte("secret"), nil)
```

Comment:
we are using `[]byte("secret")` in multiple places in this file. Declare this variable at one place and use it all other places

Reply:
Fixed